website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

```diff
@@ -13,7 +13,7 @@ Navigate to the settings page of an environment, then click **Edit**. Click the
 
 ### Keep on latest version
 
-By choosing to **Keep on latest version**, you always get the latest fixes and early access to new functionality for your dbt project. dbt Labs will handle upgrades for you, as part of testing and redeploying the dbt Cloud SaaS application.
+By choosing to **Keep on latest version**, you always get the latest fixes and early access to new functionality for your dbt project. dbt Labs will handle upgrades for you, as part of testing and redeploying the dbt Cloud SaaS application. Keep on latest version always includes the most recent version of dbt Core, and more.
```

Comment:
do we think this statement might benefit from an example? so: Keep on latest version always includes the most recent version of dbt Core, and more. For example, if the latest version is 1.8, then selecting "Keep on latest version" mean your environment is automatically updated to dbt v1.8.